Package.swift

```diff
@@ -1,44 +1,44 @@
-// swift-tools-version: 5.10
+// swift-tools-version: 6.0
 
 import PackageDescription
 
 let package = Package(
-    name: "MigrationGuide",
-    platforms: [
-        .macOS(.v10_15),
-        .iOS(.v13),
-        .watchOS(.v6),
-        .tvOS(.v13),
-        .macCatalyst(.v13),
-        .visionOS(.v1),
-    ],
-    products: [
-        .library(
-            name: "FullyMigratedModule",
-            targets: [
-                "FullyMigratedModule",
-                "MigrationInProgressModule",
-            ]
-        ),
-    ],
-    targets: [
-        .target(
-            name: "FullyMigratedModule"
-        ),
+	name: "MigrationGuide",
```

Comment:
nitpick about the tabs, especially since mixing tabs and spaces in same file.

Stick to spaces? No strong feels about their number but let's not mix spaces and tabs :)

Reply:
I too prefer spaces :)

Reply:
I figured this would come up so I just quickly wanted to elaborate on why.

I used to be strongly pro-spaces. But I was one day informed that tabs provides a major quality-of-life impovement for low-vision readers that use large font sizes. So I began making a consious effort to switch.

I screw this up all the time, however, because I always reformat to spaces for DocC content, as it does not handle semantic indentation well. While I use tabs for all my own open source stuff, I regularly get contributions that use spaces and I do not nitpick over whitespace. I assume that'll be a problem here too. FWIW, TSPL uses 4-space indentation.

I've currently reformatted the code to at least be consistent. But, I am very happy to change, since there are pretty legimate technical hurdles here, on top of preferences.